autoware_launch/launch/autoware.launch.xml

```diff
@@ -63,50 +63,27 @@
 
   <!-- System -->
   <group if="$(var launch_system)">
-    <include file="$(find-pkg-share autoware_launch)/launch/components/tier4_system_component.launch.xml">
-      <arg name="run_mode" value="$(var system_run_mode)"/>
-      <arg name="launch_system_monitor" value="$(var launch_system_monitor)"/>
-      <arg name="sensor_model" value="$(var sensor_model)"/>
-    </include>
+    <include file="$(find-pkg-share autoware_launch)/launch/components/tier4_system_component.launch.xml"/>
   </group>
 
   <!-- Map -->
   <group if="$(var launch_map)">
-    <include file="$(find-pkg-share autoware_launch)/launch/components/tier4_map_component.launch.xml">
-      <arg name="lanelet2_map_path" value="$(var map_path)/$(var lanelet2_map_file)"/>
-      <arg name="pointcloud_map_path" value="$(var map_path)/$(var pointcloud_map_file)"/>
-    </include>
+    <include file="$(find-pkg-share autoware_launch)/launch/components/tier4_map_component.launch.xml"/>
   </group>
 
   <!-- Sensing -->
   <group if="$(var launch_sensing)">
-    <include file="$(find-pkg-share tier4_sensing_launch)/launch/sensing.launch.xml">
-      <arg name="launch_driver" value="$(var launch_sensing_driver)"/>
```

Comment:
@takayuki5168 I have one question. 
Why do you delete launch_driver argument?
In this case, how do you pass "launch_driver" to sensing.launch.xml?

Reply:
Sorry, I changed wrong.
I was planning to use https://github.com/autowarefoundation/autoware_launch/blob/main/autoware_launch/launch/components/tier4_sensing_component.launch.xml to call sensing.launch.xml.

I created a fix PR. Could you review and try it?
https://github.com/autowarefoundation/autoware_launch/pull/227